.clang-format

```diff
@@ -5,35 +5,106 @@
 
 BasedOnStyle: "LLVM"
```

Comment:
If you used that, you only need to specify the values that are different than the style. See [here](https://github.com/Rombur/mfmg/blob/docker/.clang-format)

Reply:
Sure. It probably makes sense to just remove `BasedOnStyle` in the end. Currently, we have changes in nearly all the options.